wp-blocks/button.css

```diff
@@ -2,26 +2,7 @@
 	text-align: center;
 }
 
-.wp-block-button .wp-block-button__link,
-.editor-styles-wrapper .wp-block-button .wp-block-button__link {
-	transition: 0.5s;
-	padding: calc(0.32em + 2px) calc(2.5em + 2px);
-	font-size: var(--wp--preset--font-size--base);
-}
-
-/*
-	All button state changes are animated
-*/
-.wp-element-button .wp-block-button__link:focus,
-.wp-element-button .wp-block-button__link:hover,
-.wp-element-button .wp-block-button__link:active {
-	transition: all 0.25s ease-in-out;
-}
-
-.wp-block-button > a {
-	text-decoration: none;
-}
-
+.wp-block-button > a,
```

Comment:
1 suggestion here @bjcooper . Can you add a DocBlock or comment to this file that the styles were moved to the component stylesheet so that the block, plugins, and custom buttons can share all the styles.